packages/ocs/storage-pool/StoragePoolListPage.tsx

```diff
@@ -133,57 +149,65 @@ const BlockPoolList: React.FC<BlockPoolListProps> = (props) => {
         id: tableColumnInfo[2].id,
       },
       {
-        title: t('Replicas'),
+        title: t('Volume Type'),
```

Comment:
other column header are of same format...
```suggestion
        title: t('Volume type'),
```

Reply:
Done.